test/utils/nodes/baseload.go

```diff
@@ -62,18 +63,19 @@ func GetLoad(k8sCli *kubernetes.Clientset, nodeName string) (Load, error) {
 			}
 		}
 	}
+
 	// get full cpus, and always take even number of CPUs
-	cpuMillis, _ := cpu.AsInt64()
 	// we round the CPU consumption as expressed in millicores (not entire cores)
 	// in order to (try to) avoid bugs related to integer division
 	// int64(2900 / 1000) -> 2 -> roundUp(2, 2) -> 2 (correct, but unexpected!)
 	// OTOH
-	// roundUp(2900, 2000) -> 4000 -> 4000/1000 -> 4 (intended behaviour)
+	// roundUp(2900, 2000) -> 4000 -> 4000/1000 -> 4 (intended behaviour).
+	// Value() round up the millis and roundUp rounds it up to multiples of 2 if needed.
 	// TODO: we can use some testing of the test utilities here (!)
-	cpuAmount := roundUp(cpuMillis, 2000)
-	nl.CPU = *resource.NewQuantity(cpuAmount/1000, resource.DecimalSI)
-
+	klog.Infof(fmt.Sprintf("Total resources' requests by pods on node %s: CPU=%s ; mem=%s ", nodeName, cpu, mem))
```

Comment:
we're too deep in the flow for this kind of logging, better to move it elsewhere